embodiedscan/structures/bbox_3d/euler_depth_box3d.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) OpenRobotLab. All rights reserved.
+import numpy as np
+import torch
+from mmcv.ops import points_in_boxes_all, points_in_boxes_part
+
+from ..points.base_points import BasePoints
+from .euler_box3d import EulerInstance3DBoxes
+
+
+class EulerDepthInstance3DBoxes(EulerInstance3DBoxes):
+    """3D boxes of instances in Depth coordinates.
```

Comment:
We keep the "Depth" coordinate system definition in MMDet3D just for clarification of the points coordinates and the flipping augmentation.